src/actions/ast.js

```diff
@@ -9,13 +9,71 @@ import {
   getPreview
 } from "../selectors";
 
+import { ensureParserHasSourceText } from "./sources";
+
 import { PROMISE } from "../utils/redux/middleware/promise";
 import * as parser from "../utils/parser";
 
-import type { SourceId } from "debugger-html";
+import { isGeneratedId } from "devtools-source-map";
+
+import type { Source, SourceId } from "debugger-html";
 import type { ThunkArgs } from "./types";
 import type { AstLocation } from "../utils/parser";
 
+async function _fixExpression(
```

Comment:
`fix` is a bit ambiguous -- it can mean to fasten, to set into place, or it can mean to correct. correct is also problematic because it suggests that there has been some sort of corruption of the data -- perhaps better is `getSourcemapedExpression` ?